### ISSUE TITLE
How I can Set body || headerBar Background color to Transparent ? 😟

### ISSUE THREAD

**Issue by kawan2019** [dbenitez-bcn/solid_bottom_sheet#34]
How I can Set body || headerBar Background color to Transparent ? 😟

**Comment by dbenitez-bcn** [comment]
It should be transparent already. Specify your widget background to transparent

**Comment by fch21** [comment]
Solved in this other issue #29

**Comment by luscas** [comment]
```dart
SolidBottomSheet(
  headerBar: const SizedBox(), // Without headerBar
  body: Container(
    child: Center(
      child: Text('Hello World')
    )
  ),
)
```
